src/main/java/persistence/sql/ddl/EntityManager.java

```diff
@@ -0,0 +1,9 @@
+package persistence.sql.ddl;
+
+public interface EntityManager {
+    <T> T find(Class<T> clazz, Object id);
+
+    <T> Object persist(T entity);
```

Comment:
```suggestion
    <T> T persist(T entity);
```
1. 실제로 `EntityManager`는 persist의 반환 타입이 void인데요. 객체를 반환하도록 해주신 이유가 있을까요?
2. 객체를 반환한다면 `Object`가 아닌 제네릭 타입을 반환하는 것이 더 적절하지 않을까요?

Reply:
1. 미션 페이지에서 오프젝트를 반환하는것도 되어있어서, 그 방향으로 해봤습니다~
2. 오 이 부분 수정하면서 빼먹은것 같네요.